ietf-microwave-radio-link.yang

```diff
@@ -60,9 +60,10 @@ module ietf-microwave-radio-link {
   // replace the revision date with the module publication date
   // the format is (year-month-day)
 
-  revision 2024-03-03 {
+  revision 2024-09-11 {
     description
-      "pretty printing";
+      "Draft - Issue #35 (grouping common-microwave-properties
+               Issue #31 added minimum-nominal-power";
```

Comment:
According to RFC8407 (and RFC8407-bis), we need only one revision statement describing the YANG model revision in RFC8561-bis and not multiple revisions for each draft version

Reply:
noted, and will be fixed in subsequent commit